app/server/utils/misc_tasks.py

```diff
@@ -0,0 +1,8 @@
+from server import celery_app
+
+class MiscTasker(object):
```

Comment:
moving all async tasks into classes to aid with monkeypatching during tests (as above)

Reply:
not a fan of throwing random things together under the category of misc :/

Reply:
yeah good point. what do you thing makes more sense? it technically means "all async tasks that aren't blockchain ones"